docs/htdocs/info/docs/api/variation/variation_schema.html

```diff
@@ -1194,15 +1207,12 @@ <h2>Genotype tables</h2>
     <img src="/i/16/plus-button.png" class="sql_schema_icon" alt="show"/> Show species
   </a></p>
     <div id="sp_compressed_genotype_var" style="display:none;">      <ul>
-        <li class="sql_schema_species_name">Gallus gallus</li>
         <li class="sql_schema_species_name">Nomascus leucogenys</li>
         <li class="sql_schema_species_name">Ornithorhynchus anatinus</li>
         <li class="sql_schema_species_name">Ovis aries rambouillet</li>
         <li class="sql_schema_species_name">Ovis aries</li>
         <li class="sql_schema_species_name">Pan troglodytes</li>
-        <li class="sql_schema_species_name">Pongo abelii</li>
         <li class="sql_schema_species_name">Saccharomyces cerevisiae</li>
-        <li class="sql_schema_species_name">Taeniopygia guttata</li>
```

Comment:
How is this list generated? Some of these species (like _Pan troglodytes_ and _Nomascus leucogenys_) have empty `compressed_genotype_var` tables.